test_scons_local.sh

```diff
@@ -66,6 +66,10 @@ set +e
 command -v clang-format >> /dev/null 2>&1
 if [ "${PIPESTATUS[0]}" -eq 0 ]; then
     set -e
+    directory=$(pwd)
+    site_scons="${directory}/test/tool/site_scons"
+    trap 'rm -f "${site_scons}"' EXIT
+    ln -s "${directory}" "${site_scons}"
     scons -C test/tool -f SConstruct
 fi
```

Comment:
Can you also remove the link after the test?
Just leaving it in the workspace may be causing us problems.
Any way to make sure that the link is removed even if the test crashes?

Reply:
That is what the trap line does

Reply:
Missed that when I looked before.